index.html

```diff
@@ -6,7 +6,37 @@
 <p>18F Guides is the repository for best practices across our teams. 
 Our guides are open source, and you're free to use them as you wish.
 Our hope is that other digital service teams — both inside and outside the government — 
-will adopt or modify the practices outlined here. </p> 
+will adopt or modify the practices outlined here.</p>
+
+<p>Here's a curated list of guides found both here and elswhere that will help you get started:</p>
+
+##Management
```

Comment:
Whoops, just remembered that the Guides Template guides the reader to just use the `./go update_nav` script to produce the `navigation:` content. At any rate, you can view the `_config.yml` file in https://github.com/18F/guides-template/ as a model. Just note that you should have `internal: false` for any external links.